src/data/commands.js

```diff
@@ -19,3 +20,50 @@ export function load(dataName) {
         chatScene.writeText('The test data "' + dataName + '" does not exist.');
     }
 }
+
+export function unlockLevel(dataName) {
+    let chatScene = characterData.getScene(CONSTANTS.SCENES.CHAT);
+
+    const constName = dataName.toUpperCase();
+    if (constName == "ALL") {
+        // unlock all levels
+        for (let level in characterData.characterData.levels) {
+            characterData.setQuestCompleted(CONSTANTS.PREREQUISITES[level]);
+        }
+    } else if (characterData.characterData.levels.hasOwnProperty(constName)) {
+        // unlock the level if it exists and start the newly unlocked scene
+
+        characterData.setQuestCompleted(CONSTANTS.PREREQUISITES[constName]);
+        chatScene.writeText(
+            "Unlocked level: " + Utilities.prettyPrintConstant(constName),
+            FONTS.ITEM_STATS
+        );
+
+        characterData
+            .getScene(characterData.getCurrentLevel())
+            .scene.start(CONSTANTS.SCENES[constName]);
+    } else {
+        chatScene.writeText('The level "' + dataName + '" does not exist.');
```

Comment:
Not a requirement but just in case you don't know, js has string interpolation like this
```suggestion
        chatScene.writeText(`The level "${dataName}" does not exist.`);
```

Reply:
For some reason this did not work. Not sure if it is a version issue?

Reply:
`The level '${dataName}' does not exist.`

Reply:
https://github.com/EthanVieira/rs-clicker/commit/232f259eafde9d67604a79f6f7282ad5bd135377